api-metastore/src/main/java/org/zalando/nakadi/service/SubscriptionService.java

```diff
@@ -245,14 +246,16 @@ public void deleteSubscription(final String subscriptionId)
     }
 
     public ItemsWrapper<SubscriptionEventTypeStats> getSubscriptionStat(final String subscriptionId,
-                                                                        final StatsMode statsMode)
+                                                                        final StatsMode statsMode,
+                                                                        final Span span)
             throws InconsistentStateException, NoSuchEventTypeException,
             NoSuchSubscriptionException, ServiceTemporarilyUnavailableException {
         final Subscription subscription;
         try {
             subscription = subscriptionRepository.getSubscription(subscriptionId);
             authorizationValidator.authorizeSubscriptionView(subscription);
         } catch (final ServiceTemporarilyUnavailableException ex) {
+            TracingService.logErrorInSpan(span, ex.getMessage());
```

Comment:
In case if ex.getMessage() is Null for some reason, then the error won't be logged (I know that everyone is using it like that), maybe it makes sense to add method `TracingService.logErrorInSpan(span, ex)` that will not have this problem.

Reply:
From the docs for the log function:
``` 
key:value log fields. Tracer implementations should support String, numeric, and boolean values; some may also support arbitrary Objects
```
For fixing logging in the case that you suggested, I added a function that would log the exceptions' classname + optionally the localized message in https://github.com/zalando/nakadi/pull/1196/commits/e67181f6e343ceceda6b507e96e8e8f993f7c642

Reply:
Yep, that's almost exactly what I meant. I thought about `ex.getClass().getSimpleName() + ex.getMessage()`, but it's anyways good.

Reply:
Got it, thanks!